src/apprt/gtk-ng/ipc.zig

```diff
@@ -0,0 +1,3 @@
+//! functions for handling IPC
+
+pub const newWindow = @import("ipc/new_window.zig").newWindow;
```

Comment:
Given our `App.zig` is just a thin layer now, I think we can move this directly into App.zig. This feels like a lot of ceremony to forward a function declaration a couple times.